clang/test/Analysis/Checkers/WebKit/uncounted-lambda-captures.cpp

```diff
@@ -63,6 +63,18 @@ template<typename Out, typename... In> Function<Out(In...)> adopt(Detail::Callab
     return Function<Out(In...)>(impl, Function<Out(In...)>::Adopt);
 }
 
+template <typename KeyType, typename ValueType>
+class HashMap {
+public:
+  HashMap();
+  HashMap([[clang::noescape]] const Function<ValueType()>&);
```

Comment:
Should we also have a similar test case for a non-member function?

Reply:
A good point! Added a test case for static function & non-member function.